drivers/flash/CMakeLists.txt

```diff
@@ -154,6 +154,7 @@ zephyr_library_include_directories_ifdef(
   ${ZEPHYR_BASE}/drivers/memc
 )
 
+zephyr_library_sources_ifdef(CONFIG_FLASH_QSPI_RENESAS_RA flash_qspi_renesas_ra.c)
```

Comment:
Shouldn't it be after https://github.com/zephyrproject-rtos/zephyr/pull/78959/files#diff-2622b0ce0ec8ab10c395ae4c3ca5e6aa64d3e8943d6dd340ca7d2f6921fb9c2cR38 ?